TelegramBot/handlers/user_menu.py

```diff
@@ -0,0 +1,28 @@
+from telegram import KeyboardButton
+from telegram import ReplyKeyboardMarkup
+
+from TelegramBot.client.client import RestClient, bot_client
+
+
+BUTTON_REGISTER = "Register"
+BUTTON_SHOW_PRODUCTS = "Go to products"
+BUTTON_SHOW_MY_ORDERS = "Show my orders"
+BUTTON_PROFILE = "My profile"
+
+
+def get_base_reply_keyboard():
+    """
+    Register keyboard buttons into the menu
+    """
+    keyboard = [
+        [
```

Comment:
that's good, but what would be better - if buttons show up if the order is made, address is created etc...

Reply:
can do it as issue for the next sprint